packages/ai/core/generate-text/smooth-stream.test.ts

```diff
@@ -1,5 +1,5 @@
 import { describe, expect, it } from 'vitest';
-import { convertArrayToReadableStream } from '../../test';
+import { convertArrayToReadableStream } from '../../test/index';
```

Comment:
this is strange - can we directly import (from provider utils or the file)?

Reply:
this is so that the imports `test.d.ts` and `test/index.ts` don't clash, otherwise it'll pickup the wrong file

Reply:
can't you replace it with:

```
import { convertArrayToReadableStream } from '@ai-sdk/provider-utils/test';
```

Reply:
ideally for imports we avoid barrel files as much as possible and go for the original source

Reply:
will do 🙏